Makefile

```diff
@@ -0,0 +1,40 @@
+
+release: pysense pysense2 
+
+
+pysense:
+	mkdir pysense
+	mkdir pysense/lib
+	# sensors
+	cp shields/lib/LIS2HH12.py pysense/lib/
+	cp shields/lib/LTR329ALS01.py pysense/lib/
+	cp shields/lib/MPL3115A2.py pysense/lib/
+	cp shields/lib/SI7006A20.py pysense/lib/
+	# pycoproc
+	cp shields/lib/pycoproc_1.py pysense/lib/pycoproc.py
+	# example
+	cp shields/pysense_1.py pysense/main.py
+	zip -r pysense.zip pysense
```

Comment:
I would prefer if each rule cleans up it's temp folder

Reply:
I feel like people might want to use the temp example folder as a project, instead of having to manually unzip the folder